src/runtime/virtcontainers/network.go

```diff
@@ -9,19 +10,12 @@ import (
 	"context"
 	cryptoRand "crypto/rand"
 	"fmt"
-	"math/rand"
 	"net"
 	"os"
-	"os/exec"
-	"runtime"
-	"sort"
-	"time"
 
-	"github.com/containernetworking/plugins/pkg/ns"
 	"github.com/sirupsen/logrus"
 	"github.com/vishvananda/netlink"
```

Comment:
It seems that you removed *some* of the netlink utilization here -- we still have netlink included for some of the structure definitions which still sit in this file?